### ISSUE TITLE
include debug information

### ISSUE THREAD

**Issue by ohhmm** [RetroShare/RetroShare#506]
It crashes and hangs, please include debug information so we could debug it quickly.

**There is some callstack at MacOS for example crash on exit:**
Thread 0:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib          0x00007fff9b16c532 __connect + 10
1   com.yourcompany.RetroShare06    0x00000001005e8cfd connecthostport + 589
2   com.yourcompany.RetroShare06    0x00000001005e51d1 simpleUPnPcommand2 + 705
3   com.yourcompany.RetroShare06    0x00000001005e5288 simpleUPnPcommand + 24
4   com.yourcompany.RetroShare06    0x00000001005e7186 UPNP_DeletePortMapping + 198
5   com.yourcompany.RetroShare06    0x00000001008fb94e RemoveRedirect + 206
6   com.yourcompany.RetroShare06    0x00000001008fc645 upnphandler::shutdown_upnp() + 485
7   com.yourcompany.RetroShare06    0x00000001008fd4c5 upnphandler::shutdown() + 21
8   com.yourcompany.RetroShare06    0x00000001007433d4 p3NetMgrIMPL::netAssistFirewallShutdown() + 132
9   com.yourcompany.RetroShare06    0x00000001007432f0 p3NetMgrIMPL::shutdown() + 128
10  com.yourcompany.RetroShare06    0x00000001007b0bb1 RsServer::rsGlobalShutDown() + 145
11  com.yourcompany.RetroShare06    0x0000000100006f0d main + 9613
12  com.yourcompany.RetroShare06    0x0000000100004954 start + 52

But there is no symbols loaded for MS Windows. Some crash of VOIP plugin:
![image](https://cloud.githubusercontent.com/assets/190779/18641200/3e08474e-7ea4-11e6-994d-37d4b0576b64.png)

Plugins crash should not affect whole progam. 

How far we go with such debug information? It might took less time to debug if releases had full debug information.

**Comment by defnax** [comment]
please use the latest version for macos or windows, and voip plugin is not stable.

**Comment by ohhmm** [comment]
What I meant is to add installation option to install with symbols.
Or always install symbols.
Why not, it's opensource anyway.

**Comment by defnax** [comment]
you mean as Debug build?
 @thunder2

**Comment by ohhmm** [comment]
Yes, RelWithDebInfo to be exact. I think it should be official public release build
